.github/workflows/ci-cd.yml

```diff
@@ -41,9 +41,12 @@ jobs:
         os:
         - ubuntu-24.04
         - ubuntu-24.04-arm
-        experimental:
-        - false
+        # httpx=false for all runs, but then add a job on 3.13 with httpx
+        no-httpx: [false]
```

Comment:
Could you turn this into a "multi-line array" or whatever it's called. Just to keep consistent formatting.